source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -466,11 +470,14 @@ Allowed configuration combinations
 Drivers MUST enforce:
 
 * TopologyType Single cannot be used with multiple seeds.
-* ``directConnection=true`` cannot be used with multiple seeds.
+* ``directConnection=true`` and ``loadBalanced=true`` cannot be
```

Comment:
Same comment about avoiding duplication. We discuss `directConnection` validation here because there's no spec for it, but the new info is already expressed in the LB spec.

Reply:
Removed.